mapping/enable/img_tile_manager.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import division
+
+from io import BytesIO
+import os
+import os.path as op
+
+import numpy as np
+from PIL import Image
+
+from traits.api import Int, String, provides
+
+from .cacheing_decorators import lru_cache
+from .i_tile_manager import ITileManager
+from .tile_manager import TileManager
+
+
+@provides(ITileManager)
+class ImageTileManager(TileManager):
+
+    lod_dir = String
+    _num_levels = Int(0)
+
+    @lru_cache(maxsize=256)
+    def get_tile(self, zoom, row, col):
+        if zoom >= self._num_levels:
+            return None
+
+        zoom_dir = op.join(self.lod_dir, str(int(zoom)))
+        tile_path = op.join(zoom_dir, '{}.{}.npy'.format(row, col))
+        if not op.exists(tile_path):
+            return None
+
+        tile = np.load(tile_path)
+        img = Image.fromarray(tile[::-1], mode='RGB')
+        data = BytesIO()
+        img.save(data, format='png')
+        return self.process_raw(data.getvalue())
+
+    def get_tile_size(self):
+        return 256
+
+    def get_wrap_flags(self):
+        return False, False
+
+    def get_image_limits(self):
+        max_ext = 2 ** (self._num_levels - 1) * self.get_tile_size()
+        return max_ext, max_ext
```

Comment:
This doesen't seem right..

Reply:
This is a future point for refactoring. Currently, `ITileManager` doesn't have any way to communicate its boundaries. It pretends to be infinite in every direction.

This change is a small step in the direction of a solution, but I think it will require some tinkering in the canvas as well.